torchmultimodal/diffusion_labs/models/adm_unet/adm.py

```diff
@@ -24,25 +24,55 @@
     SinusoidalPositionEmbeddings,
 )
 
+DEFAULT_EMBED_NAME = "clip_image"
```

Comment:
I think this should be a parameter for ADMUnet instead of a global variable. If you look at the transform and adapter classes they all take the conditional input key as "*_field", for this case it could be "conditional_field". Also I don't think the default name should be based on CLIP as that's a dalle input and not generic to ADM. It should be something generic like "context" or "condition".

Reply:
Updated to context, left clip_image in the builder. Lmk if that's what you have in mind

Reply:
CLIP is a dalle2 input and is not specific to ADM. So I think we should remove the name from everywhere. Also I still don't believe this should be a global variable.

Reply:
Ditched the global variable and used "context" for default in both the ADMUNet class and the builder